server/migrations/community/c13819c566e7_migrate_away_from_jsonb_columns.py

```diff
@@ -0,0 +1,387 @@
+"""Migrate away from jsonb columns in project and project_version tables, introduce new file history table
+
+Revision ID: c13819c566e7
+Revises: 07f2185e2428
+Create Date: 2024-06-06 16:05:59.565541
+
+"""
+
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision = "c13819c566e7"
+down_revision = "07f2185e2428"
+branch_labels = None
+depends_on = None
+
+
+push_change_type = postgresql.ENUM(
+    "create", "update", "delete", "update_diff", name="push_change_type"
```

Comment:
this was requirement of MartinD (should be in design document) to start naming changes properly - but yes in case of diff the information is there (by checking diff is not null)